### ISSUE TITLE
Release 4.7.0 - Alpha 2 - Footprint Metrics - ALL-EXCEPT-LOGCOLLECTOR (2.5d)

### ISSUE THREAD

**Issue by wazuhci** [wazuh/wazuh#20065]
## Footprint metrics information
|                                  |                                            |
|---------------------------------|--------------------------------------------|
| **Main release stage issue #** | #20019                                         |
| **Main footprint metrics issue #** | #20020                                         |
| **Version** | 4.7.0                                      |
| **Release stage #** | Alpha 2                                         |
| **Tag** | https://github.com/wazuh/wazuh/tree/v4.7.0-alpha2 |

### Packages used
- Repository: `packages-dev.wazuh.com`
- Package path: `pre-release`
- Package revision: `1`

- **Jenkins build**: https://ci.wazuh.info/job/Test_stress/4565/

---

<details><summary>Manager</summary>

+ <details><summary>Plots</summary>

    ![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_CPU.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_Disk.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_Disk_Read.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_Disk_Written.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_FD.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_PSS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_Read_Ops.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_RSS_MAXMIN.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_RSS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_SWAP.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_USS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_VMS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/monitor-manager-Test_stress_B4565_manager-pre-release_Write_Ops.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/Test_stress_B4565_manager_analysisd_events_Decoded_events.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/Test_stress_B4565_manager_analysisd_events_Dropped_events.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/Test_stress_B4565_manager_analysisd_events_EDPS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/Test_stress_B4565_manager_analysisd_events_Written_stats.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/Test_stress_B4565_manager_analysisd_state_Number_Events.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/plots/Test_stress_B4565_manager_analysisd_state_Queues_state.png)

    </details>
+ <details><summary>Logs and configuration</summary>

    [ossec_Test_stress_B4565_manager_2023-11-06.zip](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/logs/ossec_Test_stress_B4565_manager_2023-11-06.zip)

    </details>
+ <details><summary>CSV</summary>

    [monitor-manager-Test_stress_B4565_manager-pre-release.csv](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/data/monitor-manager-Test_stress_B4565_manager-pre-release.csv)
[Test_stress_B4565_manager_analysisd_events.csv](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/data/Test_stress_B4565_manager_analysisd_events.csv)
[Test_stress_B4565_manager_analysisd_state.csv](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/data/Test_stress_B4565_manager_analysisd_state.csv)
[Test_stress_B4565_manager_remoted_state.csv](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_manager_centos/data/Test_stress_B4565_manager_remoted_state.csv)

    </details>

</details>

<details><summary>Centos agent</summary>

+ <details><summary>Plots</summary>

    ![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_CPU.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_Disk.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_Disk_Read.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_Disk_Written.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_FD.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_PSS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_Read_Ops.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_RSS_MAXMIN.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_RSS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_SWAP.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_USS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_VMS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/monitor-agent-Test_stress_B4565_centos-pre-release_Write_Ops.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/Test_stress_B4565_centos_agentd_state_AgentD_Number_of_events_buffered.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/Test_stress_B4565_centos_agentd_state_AgentD_Number_of_generated_events.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/Test_stress_B4565_centos_agentd_state_AgentD_Number_of_messages.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/plots/Test_stress_B4565_centos_agentd_state_AgentD_Status.png)

    </details>
+ <details><summary>Logs and configuration</summary>

    [ossec_Test_stress_B4565_centos_2023-11-06.zip](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/logs/ossec_Test_stress_B4565_centos_2023-11-06.zip)

    </details>
+ <details><summary>CSV</summary>

    [monitor-agent-Test_stress_B4565_centos-pre-release.csv](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/data/monitor-agent-Test_stress_B4565_centos-pre-release.csv)
[Test_stress_B4565_centos_agentd_state.csv](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_centos/data/Test_stress_B4565_centos_agentd_state.csv)

    </details>

</details>

<details><summary>Ubuntu agent</summary>

+ <details><summary>Plots</summary>

    ![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_CPU.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_Disk.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_Disk_Read.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_Disk_Written.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_FD.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_PSS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_Read_Ops.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_RSS_MAXMIN.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_RSS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_SWAP.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_USS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_VMS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/monitor-agent-Test_stress_B4565_ubuntu-pre-release_Write_Ops.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/Test_stress_B4565_ubuntu_agentd_state_AgentD_Number_of_events_buffered.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/Test_stress_B4565_ubuntu_agentd_state_AgentD_Number_of_generated_events.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/Test_stress_B4565_ubuntu_agentd_state_AgentD_Number_of_messages.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/plots/Test_stress_B4565_ubuntu_agentd_state_AgentD_Status.png)

    </details>
+ <details><summary>Logs and configuration</summary>

    [ossec_Test_stress_B4565_ubuntu_2023-11-06.zip](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/logs/ossec_Test_stress_B4565_ubuntu_2023-11-06.zip)

    </details>
+ <details><summary>CSV</summary>

    [monitor-agent-Test_stress_B4565_ubuntu-pre-release.csv](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/data/monitor-agent-Test_stress_B4565_ubuntu-pre-release.csv)
[Test_stress_B4565_ubuntu_agentd_state.csv](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_ubuntu/data/Test_stress_B4565_ubuntu_agentd_state.csv)

    </details>

</details>

<details><summary>Windows agent</summary>

+ <details><summary>Plots</summary>

    ![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_CPU.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_Disk.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_Disk_Read.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_Disk_Written.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_Handles.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_Read_Ops.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_RSS_MAXMIN.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_RSS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_USS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_VMS.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/monitor-winagent-Test_stress_B4565_windows-pre-release_Write_Ops.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/Test_stress_B4565_windows_agentd_state_AgentD_Number_of_events_buffered.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/Test_stress_B4565_windows_agentd_state_AgentD_Number_of_generated_events.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/Test_stress_B4565_windows_agentd_state_AgentD_Number_of_messages.png)
![](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/plots/Test_stress_B4565_windows_agentd_state_AgentD_Status.png)

    </details>
+ <details><summary>Logs and configuration</summary>

    [ossec_Test_stress_B4565_windows_2023-11-06.zip](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/logs/ossec_Test_stress_B4565_windows_2023-11-06.zip)

    </details>
+ <details><summary>CSV</summary>

    [monitor-winagent-Test_stress_B4565_windows-pre-release.csv](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/data/monitor-winagent-Test_stress_B4565_windows-pre-release.csv)
[Test_stress_B4565_windows_agentd_state.csv](https://ci.wazuh.com/data/Test_stress/pre-release/4.7.0/B4565-3600m/B4565_agent_windows/data/Test_stress_B4565_windows_agentd_state.csv)

    </details>

</details>

<details><summary>macOS agent</summary>

+ <details><summary>Plots</summary>

    

    </details>
+ <details><summary>Logs and configuration</summary>

    

    </details>
+ <details><summary>CSV</summary>

    

    </details>

</details>

<details><summary>Solaris agent</summary>

+ <details><summary>Plots</summary>

    

    </details>
+ <details><summary>Logs and configuration</summary>

    

    </details>
+ <details><summary>CSV</summary>

    

    </details>

</details>

## Notes
https://github.com/wazuh/wazuh-jenkins/issues/5878 used as reference

#### New issues

- https://github.com/wazuh/wazuh/issues/20102
- https://github.com/wazuh/wazuh/issues/20109
- https://github.com/wazuh/wazuh/issues/20113

#### Known issues

- https://github.com/wazuh/wazuh-jenkins/issues/4867
- https://github.com/wazuh/wazuh/issues/9311
- https://github.com/wazuh/wazuh/issues/16640
- https://github.com/wazuh/wazuh-jenkins/issues/5287
- https://github.com/wazuh/wazuh-jenkins/issues/4469
- https://github.com/wazuh/wazuh/issues/12074

**Comment by pro-akim** [comment]
# Analysis report

## Graphs :red_circle:

### Manager
PSS, RSS, USS: small increase in the number of fluctuations
Read_Ops: decrease of modulesd
VSM: small increase of remoted
EDPS: there is a fall in the first part of the test of sca but in 4.7.0, this fall does not reach values under 40 as in 4.6.0-rc2

### Windows
Disk: increase of agent.exe
- Reported: https://github.com/wazuh/wazuh/issues/20102 - New issue
RSS, USS, VMS: increase of amplitude of fluctuations
- Reported: https://github.com/wazuh/wazuh/issues/20109 - New issue
Agentd_number_of_events_buffered: flucutations between almost fixed values (between 1 and 0) while in 4.6.0 it had looked more organic (without fixed values)
- Reported: https://github.com/wazuh/wazuh/issues/20113 - New issue

### Centos
Disk: decrease of modulesd peak size
Disk_read: decrease of syscheckd
Disk_written: increase of syscheckd


### Ubuntu

## Logs :yellow_circle:


<details><summary>Manager</summary>

- Expected in stress test
```
2023/11/03 12:58:48 sca WARNING: Interval overtaken.
2023/11/03 19:00:14 wazuh-syscheckd WARNING: Real-time inotify kernel queue is full. Some events may be lost. Next scheduled scan will recover lost data.
2023/11/04 05:19:50 wazuh-modulesd:azure-logs WARNING: Interval overtaken.
2023/11/04 21:47:10 wazuh-db WARNING: After vacuum, the database '001' has become just as fragmented or worse
2023/11/03 12:57:40 wazuh-logcollector WARNING: Target 'agent' message queue is full (1024). Log lines may be lost.
2023/11/03 12:58:37 wazuh-remoted WARNING: Message queue is full (10). Events may be lost.
2023/11/03 13:46:27 wazuh-analysisd WARNING: Security Configuration Assessment decoder queue is full.
2023/11/03 18:59:24 wazuh-analysisd WARNING: Syscheck decoder queue is full.
2023/11/03 19:34:36 wazuh-analysisd WARNING: Input queue is full.
```

- Reported: https://github.com/wazuh/wazuh/issues/9311 - Known issue
```
2023/11/03 12:58:32 wazuh-modulesd:oscap ERROR: Internal error. Exiting...
```

- Reported: https://github.com/wazuh/wazuh/issues/16640 - Known issue
```
2023/11/03 12:58:32 wazuh-modulesd:vulnerability-detector WARNING: (5515): Agent '001' software could not be requested.
```

- Reported: https://github.com/wazuh/wazuh-jenkins/issues/5287 - Known issue
```
2023/11/03 19:27:33 wazuh-syscheckd WARNING: (6922): Cannot open '/tmp/syscheck_test/directories/dir700': No such file or directory
```

- Reported: https://github.com/wazuh/wazuh-jenkins/issues/4867 - Known issue
```
2023/11/03 12:58:13 wazuh-remoted WARNING: Agent key already in use: agent ID '002'
```

- Reported: https://github.com/wazuh/wazuh-jenkins/issues/4469 - Known issue
```
2023/11/03 12:58:32 wazuh-modulesd:ciscat WARNING: No evals defined. Exiting...
```

</details>



<details><summary>Windows</summary>

- Expected in stress test
```
2023/11/03 18:59:36 wazuh-agent WARNING: (6906): Real time process: no data. Probably buffer overflow.
2023/11/03 13:04:51 sca WARNING: Interval overtaken.
2023/11/03 19:04:32 wazuh-modulesd:ciscat WARNING: Interval overtaken.
```

- Reported: https://github.com/wazuh/wazuh-jenkins/issues/5287 - Known issue
```
2023/11/03 19:03:57 wazuh-agent ERROR: (6716): Could not open handle for 'c:\tmp\syscheck_test\files\fimstress.11982255'. Error code: 2
[2023-11-03_20:38:21] [ERROR] (create_delete): files\fimStress.680030 file cannot be deleted.
[2023-11-05_09:07:23] [ERROR] (create_delete): directories\dir0 directory is not a directory.
2023/11/03 19:03:57 wazuh-agent WARNING: At get_user(c:\tmp\syscheck_test\files\fimstress.11982255): CreateFile(): The system cannot find the file specified. (2)
2023/11/03 19:12:41 wazuh-agent WARNING: (6922): Cannot open 'c:\tmp\syscheck_test\directories\dir3804': No such file or directory
```

- Reported: https://github.com/wazuh/wazuh-jenkins/issues/4469 - Known issue
```
2023/11/03 15:34:33 wazuh-modulesd:ciscat ERROR: Timeout expired executing 'C:\cis-cat\benchmarks\CIS_Microsoft_Windows_Server_2016_Benchmark_v1.0.0-xccdf.xml'.
2023/11/03 12:59:01 wazuh-modulesd:ciscat ERROR: Report result file 'tmp\ciscat-report.txt' missing: No such file or directory
```

- Reported: https://github.com/wazuh/wazuh/issues/12074 - Known issue
```
2023/11/05 12:59:49 wazuh-agent ERROR: (6613): Real time Windows callback process: 'Access is denied.' (5).
```

- Reported: https://github.com/wazuh/wazuh-jenkins/issues/4867 - Known issue
```
2023/11/03 12:58:23 wazuh-agent ERROR: (1216): Unable to connect to '[172.31.7.194]:1514/tcp': 'No connection could be made because the target machine actively refused it.'.
```

</details>



<details><summary>Centos</summary>

- Expected in stress test
```
2023/11/03 12:58:43 sca WARNING: Interval overtaken.
2023/11/03 19:00:03 wazuh-syscheckd WARNING: Real-time inotify kernel queue is full. Some events may be lost. Next scheduled scan will recover lost data.
```

- Reported: https://github.com/wazuh/wazuh-jenkins/issues/4469 - Known issue
```
2023/11/03 12:58:36 wazuh-modulesd:ciscat ERROR: Report result file 'tmp/ciscat-report.txt' missing: No such file or directory
2023/11/03 12:58:36 wazuh-modulesd:ciscat ERROR: Failed reading scan results for policy '/var/ossec/wodles/cis-cat/benchmarks/CIS_Google_Chrome_Benchmark_v1.2.0-xccdf.xml'
```

- Reported: https://github.com/wazuh/wazuh/issues/9311 - Known issue
```
2023/11/03 12:58:09 wazuh-modulesd:oscap ERROR: Internal error. Exiting...
```

- Reported: https://github.com/wazuh/wazuh-jenkins/issues/4867 - Known issue
```
2023/11/03 12:58:16 wazuh-agentd ERROR: (1137): Lost connection with manager. Setting lock.
2023/11/03 12:58:16 wazuh-agentd ERROR: (1216): Unable to connect to '[172.31.7.194]:1514/tcp': 'Connection refused'.
2023/11/03 12:58:16 wazuh-modulesd WARNING: Process locked due to agent is offline. Waiting for connection...
```

</details>



<details><summary>Ubuntu</summary>

- Expected in stress test
```
2023/11/03 12:58:52 sca WARNING: Interval overtaken.
2023/11/03 19:00:14 wazuh-syscheckd WARNING: Real-time inotify kernel queue is full. Some events may be lost. Next scheduled scan will recover lost data.
```

- Reported: https://github.com/wazuh/wazuh-jenkins/issues/4469 - Known issue
```
2023/11/03 12:58:06 wazuh-modulesd:ciscat ERROR: CIS-CAT tool not found at '/var/ossec/wodles/cis-cat'.
```

- Reported: https://github.com/wazuh/wazuh-jenkins/issues/4867 - Known issue
```
2023/11/03 12:58:23 wazuh-agentd ERROR: (1216): Unable to connect to '[172.31.7.194]:1514/tcp': 'Connection refused'.
```

- Reported: https://github.com/wazuh/wazuh/issues/9311 - Known issue
```
2023/11/03 12:58:36 wazuh-modulesd:oscap ERROR: Internal error. Exiting...
```

</details>

**Comment by damarisg** [comment]
LGTM!
